data/data-pipeline/data_pipeline/tests/sources/eamlis/test_etl.py

```diff
@@ -63,7 +63,7 @@ def setup_method(self, _method, filename=__file__):
     def test_init(self, mock_etl, mock_paths):
         """Tests that the mock NationalRiskIndexETL class instance was
```

Comment:
`NationalRiskIndexETL` seems like we should remove it?

Reply:
Yeah this isn't my code -- if you fix that on your branch, great! 

I ran `black` on all .py files and it cleaned up this one.